README.md

```diff
@@ -1,24 +1,66 @@
----
-description: Introduction of Web3URL
----
+# Web3 protocol
 
-# Web3URL
+``web3://`` is a new protocol to access content returned by smartcontracts in EVM-compatible blockchains. 
```

Comment:
```suggestion
``web3://`` is a new protocol to access content returned by smart contracts in EVM-compatible blockchains. 
```

I think we should use "smart contract" instead of "smartcontract" (as used by Ethereum docs).

Reply:
Ok I'll update that everywhere!